src/chainparams.cpp

```diff
@@ -289,7 +292,7 @@ class CRegTestParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_CSV].nStartTime = 0;
         consensus.vDeployments[Consensus::DEPLOYMENT_CSV].nTimeout = 999999999999ULL;
```

Comment:
Unrelated nit, 999999999999ULL is a terribly confusing value for these nTimeouts!

Reply:
Used constants from @ajtowns's commit.